step-templates/trigger-multitenant-deployment.json

```diff
@@ -0,0 +1,75 @@

```

Comment:
```
#$debug=1

if ($debug){
	Write-Host "### Please, place here TriggerMultitenantDeployment call(s) with mock parameters' values specific to your project."
} else {
```
I don't think should this code be included in the step template

Reply:
```
$errrors=0
```
an extra `r` snuck in here

Also, why not use `$true` or `$false` instead of `0/1` as it's being used to check for a _true/false_ value

Reply:
```
			if($errrors)
			{
				Write-Error "### Encoutered an error. Interrupting the task."
				exit 1
			}
```
You can use `Fail-Step <message>` here instead of `write-error;exit 1`

Reply:
Done.